pkg/dcgmexporter/const.go

```diff
@@ -25,26 +25,28 @@ const (
 	DCGMXIDErrorsCount DCGMExporterMetric = iota + 9000
 )
 
+// DCGMFields maps DCGMExporterMetric String to enum
+var DCGMFields = map[string]DCGMExporterMetric{
```

Comment:
Are we going to export this field outside of the package? If not, let's make it private.

Reply:
Why is the DCGMFields name?  This variable doesn't represent the DCGM Field.

Reply:
Not an exported field just following GO convention ([source](https://go.dev/wiki/CodeReviewComments)), since DCGM is an acronym it could either be `dcgmFields` or `DCGMFields`. I opted for DCGMFields as most of the variables within `const.go` uses capitalized version of `DCGM`.

The reason I opted for `Fields` as the other part of the name is because within `Counter` struct it is called `Field*` ([source](https://github.com/NVIDIA/dcgm-exporter/blob/3250bfebedf1b3e21c060c5b21479f43e740a94b/pkg/dcgmexporter/types.go#L121)). Also, within `extractCounters` function I see heavy use of the term `field`.

As they say naming is hard in Computer Science, let me know what would be a better alternative.

Reply:
At the same time, `dcgm` works too [(source)](https://go.dev/wiki/CodeReviewComments#initialisms)